Library/Homebrew/test/.simplecov

```diff
@@ -9,11 +9,11 @@ SimpleCov.start do
   # tests to be dropped. This causes random fluctuations in test coverage.
   merge_timeout 86400
 
-  add_filter "/Homebrew/cask/spec/"
-  add_filter "/Homebrew/cask/test/"
-  add_filter "/Homebrew/compat/"
-  add_filter "/Homebrew/test/"
-  add_filter "/Homebrew/vendor/"
+  add_filter "/Library/Homebrew/cask/spec/"
+  add_filter "/Library/Homebrew/cask/test/"
+  add_filter "/Library/Homebrew/compat/"
+  add_filter "/Library/Homebrew/test/"
+  add_filter "/Library/Homebrew/vendor/"
```

Comment:
Why the change? In any case, this feels misplaced as this is completely unrelated to the changes in this PR (unless I'm missing something).